perses/tests/test_smc.py

```diff
@@ -0,0 +1,143 @@
+                                       neglected_new_angle_terms = fe_setup._vacuum_forward_neglected_angles,
+                                       neglected_old_angle_terms = fe_setup._vacuum_reverse_neglected_angles,
+                                       softcore_LJ_v2 = True,
+                                       interpolate_old_and_new_14s = False)
+
+zero_state_error, one_state_error = validate_endstate_energies(fe_setup._vacuum_topology_proposal,
+                                                               hybrid_factory,
+                                                               added_energy = fe_setup._vacuum_added_valence_energy,
+                                                               subtracted_energy = fe_setup._vacuum_subtracted_valence_energy,
+                                                               beta = beta,
+                                                               ENERGY_THRESHOLD = ENERGY_THRESHOLD)
+#default arguments
+lambda_protocol = 'default'
+temperature = 300 * unit.kelvin
+trajectory_directory = 'test_smc'
+trajectory_prefix = 'out'
+atom_selection = 'not water'
+timestep = 1 * unit.femtoseconds
+collision_rate = 1 / unit.picoseconds
+eq_splitting_string = 'V R O R V'
+neq_splitting_string = 'V R O R V'
+ncmc_save_interval = None
+measure_shadow_work = False
+neq_integrator = 'langevin'
+external_parallelism = None
+internal_parallelism = {'library': ('dask', 'LSF'), 'num_processes': 2}
+os.system(f"mkdir {trajectory_directory}")
+
+@nottest
+@skipIf(istravis, "Skip helper function on travis")
+def sMC_setup():
+    """
+    function to setup local sMC
+    """
+    ne_fep = SequentialMonteCarlo(factory = hybrid_factory,
+                                      lambda_protocol = lambda_protocol,
+                                      temperature = temperature,
+                                      trajectory_directory = trajectory_directory,
+                                      trajectory_prefix = trajectory_prefix,
+                                      atom_selection = atom_selection,
+                                      timestep = timestep,
+                                      eq_splitting_string = eq_splitting_string,
+                                      neq_splitting_string = neq_splitting_string,
+                                      collision_rate = collision_rate,
+                                      ncmc_save_interval = ncmc_save_interval,
+                                      external_parallelism = None,
+                                      internal_parallelism = None)
+    ne_fep.minimize_sampler_states()
+    ne_fep.equilibrate(n_equilibration_iterations = 10,
+                           n_steps_per_equilibration = 1,
+                           endstates = [0,1],
+                           decorrelate = True,
+                           timer = True,
+                           minimize = False)
+    return ne_fep
+
+def test_local_sMC():
```

Comment:
What is this test doing? Just checking that AIS will run without failures? Are there any other tests that we can make note to implement in future, like asserting that forward = - reverse?

Reply:
going to add several tests to check that each method is called is working properly